### PR TITLE
docs(button): add danger button description

### PR DESCRIPTION
**Pull request by silviowolf**
## 💡 What is the current behavior?

GitHub Issue Number: #<ISSUE NUMBER>

## 🆕 What is the new behavior?

- add danger button description
- add ellipsis description
- add min-width description

### CODE REVIEW

**Review by kathrinschalber — APPROVED** [review]

File: packages/documentation/docs/controls/buttons/_button_styleguide.md [hunk 1]
```diff
@@ -24,24 +25,28 @@ Buttons initiate actions, apply actions to selected objects and activate/deactiv
 ## Behavior in context
 - **Interaction:** Buttons can be triggered by pressing anywhere within the button container. When buttons are focused, they can be triggered by pressing `Space`.
 - **Text truncation:** Button labels are not truncated. All text on buttons is one line only. 
-- **Alignment:** Buttons can be left-justified or right-justified or fully span a container's width.
+- **Ellipsis (...):** We use ellipsis when a button action requires further user input for executing. Example: The button "Connect..." opens a modal with a list of potential connection targets from which the user selects one. This rule does not apply to actions with a subsequent confirmation dialog.
```
[the remark this answers is not in the document]
Comment (silviowolf) [reply packages/documentation/docs/controls/buttons/_button_styleguide.md:28]: The ellipsis pattern is one of the very classical standards on all usual operating systems used on buttons and menu actions ("Save" vs. "Save as…", "Print" vs. "Print…")

File: packages/documentation/docs/controls/buttons/_button_styleguide.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ Buttons initiate actions, apply actions to selected objects and activate/deactiv
 ## Variants
 
 - **Primary button:** In our applications, we most often use the primary button variant.
-- **Secondary button:** The secondary button variant has a grayscale appearance to distinguish it from the primary button. 
+- **Secondary button:** The secondary button variant has a grayscale appearance to distinguish it from the primary button.
+- **Danger button:** We use the danger button variant to emphasize particularly dangerous, destructive or critical actions that cannot easily be undone, consider this especially in confirmation dialogs.
```
Comment (nuke-ellington) [inline packages/documentation/docs/controls/buttons/_button_styleguide.md:12]:
```suggestion
- **Danger button:** We use the danger button variant to emphasize particularly dangerous, destructive or critical actions that cannot easily be undone,. Consider this especially in confirmation dialogs. 
```

File: packages/documentation/docs/controls/buttons/_button_styleguide.md [hunk 3]
```diff
@@ -24,24 +25,28 @@ Buttons initiate actions, apply actions to selected objects and activate/deactiv
...
+- Do use the danger button for destructive or critical actions like "delete" or "remove", especially in confirmation dialogs
 - Do use a default margin between adjacent buttons of at least `0.5rem`
-- Do adjust default and outline button width to label length or container width 
-- Don't extend your ghost button width beyond the label's available width
-- Don't only rely on standard buttons when many actions/functions are necessary, consider dropdown, split or menu buttons or moving some functionality to a panel or a dialog
-- Don't use buttons for navigation
+- Do adjust default and outline button width to label length or container width
+- Don’t use the danger button excessively and repetitively as in lists or tables, use it carefully and very selectively instead
+- Don’t extend your ghost button width beyond the label’s available width
+- Don’t only rely on standard buttons when many actions/functions are necessary, consider dropdown, split or menu buttons or moving some functionality to a panel or a dialog
```
Comment (nuke-ellington) [inline packages/documentation/docs/controls/buttons/_button_styleguide.md:48]:
```suggestion
- Don’t only rely on standard buttons when many actions/functions are necessary,. Consider dropdown, split or menu buttons or moving some functionality to a panel or a dialog
```

File: packages/documentation/docs/controls/buttons/_button_styleguide.md [hunk 4]
```diff
@@ -24,24 +25,28 @@ Buttons initiate actions, apply actions to selected objects and activate/deactiv
...
 
 ## States
 
-Buttons have six states: Default, hover, active, disabled, loading and focused. In a disabled state, buttons are visually displayed but don't offer any user interaction.
+Buttons have six states: Default, hover, active, disabled, loading and focused. In a disabled state, buttons are visually displayed but don’t offer any user interaction.
 
 ![Button states](https://www.figma.com/file/wEptRgAezDU1z80Cn3eZ0o/iX-Pattern-Illustrations?type=design&node-id=132-13020&mode=design&t=KAxDgJoFX436Uk0b-11)
 
-## Dos and Don'ts
+## Dos and Don’ts
```
Comment (nuke-ellington) [inline packages/documentation/docs/controls/buttons/_button_styleguide.md:39]:
```suggestion
## Dos and Don'ts
```

File: packages/documentation/docs/controls/buttons/_button_styleguide.md (hunk 3 above)
Comment (nuke-ellington) [inline packages/documentation/docs/controls/buttons/_button_styleguide.md:48]:
```suggestion
- Don't use the danger button excessively and repetitively as in lists or tables, use it carefully and very selectively instead
- Don't extend your ghost button width beyond the label’s available width
- Don't only rely on standard buttons when many actions/functions are necessary, consider dropdown, split or menu buttons or moving some functionality to a panel or a dialog
```

File: packages/documentation/docs/controls/buttons/_icon-button_styleguide.md [hunk 5]
```diff
@@ -5,17 +5,17 @@ Icon buttons are button elements containing only an icon and no text. Due to the
 1. Container
 2. Icon
 
-All the variants, options and states of the ix button component apply to the icon button. We've listed additional or deviating specifications here.
+All the variants, options and states of the [button](./button.md) component apply to the icon button. We’ve listed additional or deviating specifications here.
```
Comment (nuke-ellington) [inline packages/documentation/docs/controls/buttons/_icon-button_styleguide.md:8]:
```suggestion
All the variants, options and states of the [button](./button.md) component apply to the icon button. We've listed additional or deviating specifications here.
```

File: packages/documentation/docs/controls/buttons/_icon-button_styleguide.md [hunk 6]
```diff
@@ -5,17 +5,17 @@ Icon buttons are button elements containing only an icon and no text. Due to the
...
 
 ## Options
 - **Color:** The color of the icon displayed on an icon button is adjustable. In our applications, we only adjust the icon color when we place icon buttons on backgrounds with a non-standard color to maintain a proper contrast between the elements.
 - **Oval:** The shape of icon button containers can be adjusted from square to oval. The recommended shape of icon buttons depends on the shape of the parent component. We typically use square icon buttons within rectangular components or in a button cluster, and oval icon buttons within oval components.
 - **Size:** Icon buttons can have three different sizes. We use the extra small size (12) within very small parent components, the small size (16) within any standard parent components (e.g. to clear the search input) and the default size (24) for standalone applications.
 
-## Dos and Don'ts
+## Dos and Don’ts
```
Comment (nuke-ellington) [inline packages/documentation/docs/controls/buttons/_icon-button_styleguide.md:15]:
```suggestion
## Dos and Don'ts
```

File: packages/documentation/docs/controls/buttons/_icon-button_styleguide.md [hunk 7]
```diff
@@ -5,17 +5,17 @@ Icon buttons are button elements containing only an icon and no text. Due to the
...
 - Do use icons that have a clear meaning for the user, otherwise use text buttons
-- Don't use icon buttons in large numbers, instead use a toolbar
-- Don't stretch icon buttons to span a container's width
+- Don’t use icon buttons in large numbers, instead use a toolbar
+- Don’t stretch icon buttons to span a container’s width
```
Comment (nuke-ellington) [inline packages/documentation/docs/controls/buttons/_icon-button_styleguide.md:18]:
```suggestion
- Don't use icon buttons in large numbers, instead use a toolbar
- Don't stretch icon buttons to span a container’s width
```

**Review by nuke-ellington — CHANGES_REQUESTED** [review]

**Review by nuke-ellington — APPROVED** [review]

**Pull request merged**